src/content/docs/reference/structeezy-dynamic-fields.mdx

```diff
@@ -182,7 +182,7 @@ The `{{DATETIME}}` placeholder will be replaced by a unique date.
 ```
 
 :::note
-The date is generated at injection time. I guarantees that each time you inject a structure, the date will be different. But once injected, the date becomes static in Bricks.
+The date is generated at injection time. I guarantee that each time you inject a structure, the date will be different. But once injected, the date becomes static in Bricks.
```

Comment:
Thanks for all the fixes.
For this one, I think when I wrote this I meant: "It Guarantees" rather than "I guarantee".
```suggestion
The date is generated at injection time. It guarantees that each time you inject a structure, the date will be different. But once injected, the date becomes static in Bricks.
```